apps/web/src/scss/variables.scss

```
@@ -108,6 +108,9 @@ $darkDarkBlue2: #1f242e;
 $darkInputColor: $white;
 $darkInputPlaceholderColor: $darkGrey1;
 
+$code-color: #c01176;
+$code-color-dark: #f08dc7;
```

The variable names aren't aligned with the convention in this file. The light color should be under the light variables.
```suggestion
$lightCodeColor: #c01176;
$darkCodeColor: #f08dc7;
```